operate/qa/import-performance-tests/pom.xml

```diff
@@ -15,7 +15,8 @@
 
     <dependency>
       <groupId>io.camunda</groupId>
-      <artifactId>operate-webapp</artifactId>
+      <artifactId>camunda-zeebe</artifactId>
```

Comment:
Yes, the `dist` module is called `camunda-zeebe`. Since the `Application` class moved to the `dist` module, the integration tests now require the `dist` module. (Similar to what we have in Zeebe as well).